selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/SeleniumWebDriverHelper.java

```diff
@@ -156,45 +156,37 @@ public WebElement waitVisibility(WebElement webElement) {
    *
    * @param elementLocator
    * @param timeout waiting time in seconds
-   * @return {@code true} - if not visible,
-   *     <p>throws {@link org.openqa.selenium.TimeoutException} - if visible after timeout.
```

Comment:
`@throws {@link org.openqa.selenium.TimeoutException} - if is visible during timeout.` could be still useful for reader.